cli/src/cmd/forge/verify/mod.rs

```diff
@@ -166,6 +173,13 @@ impl figment::Provider for VerifyArgs {
 impl VerifyArgs {
     /// Run the verify command to submit the contract's source code for verification on etherscan
     pub async fn run(self) -> eyre::Result<()> {
+        if self.show_standard_json_input {
+            let args =
+                EtherscanVerificationProvider::default().create_verify_request(&self).await?;
+            println!("{}", args.source);
```

Comment:
technically, we are showing full request here which might be flattened sources or standard json input. maybe we should rename the flag and serialize before printing

Reply:
the `source` should be json here already?

Reply:
ah, true